.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,29 @@
+name: Pull Request
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        token: '${{ secrets.PAT_DECRAPIFIER }}'
+        fetch-depth: 0
+    - name: Use Node.js 16.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 16.x
+    - name: Cache Node.js modules
+      uses: actions/cache@v2
+      with:
+        # npm cache files are stored in `~/.npm` on Linux/macOS
+        path: ~/.npm
+        key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.OS }}-node-
+          ${{ runner.OS }}-
+    - run: npm ci
+    - run: npm run lint:ci || true
```

Comment:
This `|| true` is only so duti can report the results. Without duti these will silently fail. You can get rid of it here.

On that note, there is an edge case in our public repos where if the lint or test scripts fail for a reason that doesn't result in an `results.json` file, the steps will silently fail. I'll create a ticket to clean that up.

Reply:
done.